pages.id/windows/print.md

```diff
@@ -0,0 +1,12 @@
+# print
+
+> Mencetak sebuah file teks ke dalam mesin pencetak (printer)
+> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/print>.
+
+- Mencetak file teks ke dalam mesin pencetak (printer) default:
```

Comment:
Isn't there a fitting Indonesian translation fo `file`?

Reply:
`file` (as a noun) does not have an equivalent word in Indonesian, but there's `dokumen` which literally means `document`, and `mengajukan` for the verb variant of `file`.

I and other contributors (especially those who worked during Hacktoberfest) tend to use the term `file dan direktori` instead of any other else.

Reply:
So is this correct now or should it be `Mencetak file dan direktori teks ke ...`?

Reply:
No, unless if the `print` command supports printing files in a specified directory.